Kickstarter-iOS/Views/Controllers/CommentRepliesViewController.swift

```diff
@@ -183,3 +179,8 @@ private let tableViewStyle: TableViewStyle = { tableView in
     |> \.rowHeight .~ UITableView.automaticDimension
     |> \.separatorStyle .~ .none
 }
+
+internal func commentRepliesRowBehaviour(for comment: Comment,
```

Comment:
I would add some documentation to this method. it's not obvious what's happening based on the method name.